Lionel work ex 4 (#24)
mitre/fhir-exercises: Exercise_4/docs/part-2.md

### Mapping USCDI to EHR data components  
USCDI includes the following high-level classes for defining data objects.

The classes’ specific healthcare data elements are listed in the Appendix A graphic.

>* Assessment and plan of treatment
>* Care Team members
>* Clinical Notes
>* Goals
>* Health concerns
>* Immunizations
>* Laboratory
* Medications
* Patient Demographics
* Problems
* Procedures
* Provenance
* Smoking Status
* Unique device identifier
* Vital signs

![USCDI Class Components](img/uscdi_classes_1.png)

Source:  [https://fhim.org/resources/page1](https://fhim.org/resources/page1){target=_blank}

!!! note "Example of how to read USCDI"
    Remember that “social-history” observation about smoking status that we saw earlier? 
    Here Smoking Status is listed in the draft USCDI v3: [https://www.healthit.gov/isa/taxonomy/term/811/draft-uscdi-v3](https://www.healthit.gov/isa/taxonomy/term/811/draft-uscdi-v3){target=_blank}

??? question "Knowledge Check: What Data Class Is Smoking Under for USCDI Draft V3?"
    Health Status

??? question "Knowledge Check: Was this always true?"

>     No! In previous versions of USCDI it was it’s own class.

Reviewer: ```suggestion
    No! In previous versions of USCDI, Smoking Status was its own class.
```
